iOS_Sanity.py

```diff
@@ -208,17 +208,48 @@ def test_7_InsufficientFunds(self):
         okButton = self.findById('OK')
         okButton.click()
 
-    def test_8_GoodTransaction(self):
+    def test_8_LongMemo(self):
         # Verify that the address and amount fields exists
         addressField = self.findById('AddressField')
         amountField = self.findById('AmountField')
+        memoField = self.findById('MemoField')
 
         addressField.click()
         addressField.clear()
         addressField.send_keys(TestCases.qaAccount)
+        amountField.click()
         amountField.clear()
+        amountField.send_keys('350')
+        memoField.click()
+        memoField.clear()
+
+        # Memo should be restricted to 32 characters
+        memoField.send_keys('a'*50)
+
+        # Verify that the send button exists
+        sendButton = self.findById('SendButton')
```

Comment:
If it's not, we will get an exception on `click`.
If we really need to check for UI, need to `assertNotNull` on the return of `findByid` ( and if it's `null` stop the test )